dishy

```diff
@@ -111,7 +117,7 @@ WARNING: UNINSTALLATION OF DISHY MAY CAUSE SYSTEM INSTABILITY
 }
 
 dishy_lots() {
-  for i in {1..10}; do
+  for i in $(eval echo "{1..$lots_count}"); do
```

Comment:
I'd prefer `for ((i = 1; i<=$lots_count; i++); do` or `for i in $(seq 1 $lots_count); do` as better form here.